GraphQL-TSC.md

```diff
@@ -183,7 +183,7 @@ To balance preserving the voting ability of all TSC members with the desire for
 
 A quorum is a majority (more than half, or 2/3 for a supermajority vote) of the TSC *attending members*. A quorum must cast a ballot in order for a vote to be valid.
 
-A TSC *attending member* is a member who has attended one of the previous three meetings. Should a TSC member miss three consecutive meetings, they will no longer be counted when determining quorum (but may still vote). A member starts counting towards quorum as of attending a meeting.
+A TSC *attending member* is a member who has attended a TSC meeting in the past 100 days. Should a TSC member not attend a TSC meeting for more than 100 days, they will no longer be counted when determining quorum (but may still vote). A member starts counting towards quorum as of attending a TSC meeting.
```

Comment:
I like this, but I wonder if specifically framing attending member as joining these special-agenda meetings might lead to pedantic issues.

For example if first of month meeting doesn't align to your timezone and you can't make it, but second of the month does - that person would still be showing up and clearly available, even though by this definition they might not be considered as such.

My suggestion would be to just `s/TSC meeting/meeting/g` this paragraph?

Reply:
But if a member misses every other meeting it does not hot the 100 days boundary or? So he would still be an attending member.

Reply:
Ah, I see what you mean ... and you are right. Maybe any meeting should count?

Reply:
I’d be very happy to make this more broad 👍

Reply:
I've made this edit :+1: